elements/workflow/nuxeo-document-task.js

```diff
@@ -163,12 +163,22 @@ Polymer({
           </div>
           <div id="assignedActors" class="vertical spaced">
             <span>[[i18n('tasks.actors.assigned')]]</span>
-            <nuxeo-tags type="user" items="[[task.actors]]"></nuxeo-tags>
+            <template is="dom-if" if="[[_hasActorType(task.actors, 'group')]]">
+              <nuxeo-tags type="group" items="[[_getActorsByType(task.actors, 'group')]]"></nuxeo-tags>  
+            </template>
+            <template is="dom-if" if="[[_hasActorType(task.actors, 'user')]]">
+              <nuxeo-tags type="user" items="[[_getActorsByType(task.actors, 'user')]]"></nuxeo-tags>
+            </template>
```

Comment:
Can't we go with a computed binding that computes the type based on `task.actors`? Something like:
```suggestion
            <nuxeo-tags type="[[_computeTagType(task.actors)]]" items="[[task.actors]]"></nuxeo-tags>
```

Reply:
🤔 Doesn't work if you have assigned groups and individual users simultaneously. Should we care about this one?

Reply:
Ah, I understand now. We should care and this is a possibility, indeed. It seems that we are just fighting against the `nuxeo-tags` element. But I have no other suggestion, then. The alternative would be to add some sort of type resolver to the `nuxeo-tags`, but that by itself ins an improvement and has other implications to the public API of the element.

Reply:
Exactly. In fact that was my first idea, to extend the public API of the `nuxeo-tags` element by adding a type resolver. What if we keep the same public API, but extend the `type` attribute with `auto` as a possible value, and  move all the logic to inside the `nuxeo-tags`?

Reply:
So `type` could be `user`, `group` and `auto`.

Reply:
I think that makes perfect sense, but perhaps we should do it as a follow up ticket? In the end, people will still be forced to update their layouts, right? If so, we won't get any immediate benefit to committing to this new API.

Reply:
No, you're totally right, we can open an improvement ticket to add this new API, as we'd still need to update the layouts.

Reply:
FYI https://jira.nuxeo.com/browse/ELEMENTS-1467